spring-cloud-starter-stream-source-http/src/test/java/org/springframework/cloud/stream/app/http/source/HttpSourceTests.java

```diff
@@ -64,41 +64,32 @@
  */
 @RunWith(SpringRunner.class)
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-@TestPropertySource(properties = {
-		"spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration" })
 public abstract class HttpSourceTests {
 
 	@Autowired
 	protected Source channels;
 
-	@LocalServerPort
-	protected int port;
-
 	@Autowired
 	protected MessageCollector messageCollector;
 
+	@Autowired
 	protected TestRestTemplate restTemplate;
 
-	@Before
-	public void setup() {
-		this.restTemplate = new TestRestTemplate();
-	}
-
-	@TestPropertySource(properties = "http.pathPattern = /foo")
+	@TestPropertySource(properties = {
+			"http.pathPattern = /foo",
+			"spring.cloud.stream.security.enabled=false" })
```

Comment:
I don't think we need to specify this property explicitly.
We definitely have to be sure that default behavior is with the disabled security.

So, this change has to be removed.

Reply:
+1

Reply:
Because after latest updates the `core\app-starters-security-common` disables security only for Info, Health, LogFile and Bindings by default the `http` app uses the `config\application.yml` to set `spring.cloud.stream.security.enabled` to `false`